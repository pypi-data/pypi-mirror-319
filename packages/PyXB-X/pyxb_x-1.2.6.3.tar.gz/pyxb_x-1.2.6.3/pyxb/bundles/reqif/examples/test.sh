#!/bin/sh

: ${PYXB_TEST_ROOT:=${PYXB_ROOT}/tests}
. ${PYXB_TEST_ROOT}/support.sh

TEST_URI=https://raw.githubusercontent.com/redsteve/EnterpriseArchitect_ReqIF_AddIn/master/EA_ReqIF_AddIn/example.reqif.xml

if [ ! -f in.xml ]; then
  wget -O in.xml ${TEST_URI} || fail retrieving document
fi

PYXB_ARCHIVE_PATH=${PYXB_ROOT}/pyxb/bundles/reqif//:+
export PYXB_ARCHIVE_PATH

python process.py > test.out || fail running
cmp test.out test.expected || fail output comparison
passed
