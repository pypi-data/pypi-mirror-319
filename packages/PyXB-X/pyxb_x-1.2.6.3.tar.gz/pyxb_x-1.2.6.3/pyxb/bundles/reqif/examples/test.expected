Optional comment associated with the Exchange Document as a whole
